transport/tchannel/channel_outbound.go

```diff
@@ -177,6 +183,19 @@ func (o *ChannelOutbound) Call(ctx context.Context, req *transport.Request) (*tr
 	}, nil
 }
 
+// Introspect returns basic status about this outbound.
+func (o *ChannelOutbound) Introspect() introspection.OutboundStatus {
+	state := "Stopped"
+	if o.IsRunning() {
+		state = "Running"
+	}
+	return introspection.OutboundStatus{
+		Transport: "tchannel",
+		Endpoint:  o.addr,
+		State:     state,
```

Comment:
in a future version maybe we could extract some interesting info from tchannel introspection. A tchannel specific debug page will provide the full detail.